code/game/objects/items/glassjar.dm

```diff
@@ -135,7 +199,35 @@
 	STOP_PROCESSING(SSprocessing, S)
 	update_icon()
 
+/obj/item/glass_jar/gumball
+	contains = JAR_GUMBALL
+
+/obj/item/glass_jar/gumball/Initialize()
+	..()
+	for(var/i = 1 to GUMBALL_MAX)
+		var/obj/item/clothing/mask/chewable/candy/gum/gumball/G = new
+		G.forceMove(src)
+		gumballs_contained += G
+
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/item/glass_jar/gumball/LateInitialize()
+	update_icon()
+
+/obj/item/glass_jar/gumball/medical
+
+/obj/item/glass_jar/gumball/medical/Initialize()
+	for(var/i = 1 to GUMBALL_MAX)
```

Comment:
```suggestion
/obj/item/glass_jar/gumball/medical/Initialize()
	. = ..()
	for(var/i = 1 to GUMBALL_MAX)
```

Reply:
This actually causes the medical jar to spawn with 15 of the normal gumballs at the same time.